usage_example.ipynb

```diff
@@ -104,7 +112,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "collapsed": true
+   },
```

Comment:
tam gdzie się zmienia execution_count na null to jeszcze spoko, ale te zmiany z metadata to śmieci

Reply:
jeszcze w dwóch innych miejscach to jest jakby co